wafer/schedule/templates/wafer.schedule/penta_schedule.xml

```diff
@@ -20,7 +22,7 @@
     <acronym>{{ WAFER_CONFERENCE_ACRONYM }}</acronym>
   </conference>
   {% for page in schedule_pages %}
-    <day date="{{ page.block.start_time.date.isoformat }}" start="{{ page.block.start_time.isoformat }}" end="{{ page.block.end_time.isoformat }}" index="{{ forloop.counter }}">
+    <day date="{{ page.block.start_time|localtime|date:"Y-m-d" }}" start="{{ page.block.start_time|localtime|date:"c" }}" end="{{ page.block.end_time|localtime|date:"c" }}" index="{{ forloop.counter }}">
```

Comment:
Does localtime here do anything, given that 'localtime on' is already set?

Reply:
The `localtime on` I think has the effect of localizing date and time objects that attempt to render. But if it's already transformed to a string, it's too late.

Reply:
As I read the django documentation, the only difference between the locatime filter, and the 'localtime on' setting is the support for naive datetime objects, which we shouldn't have if 'load tz' is included. Do you have an example where leaving it out gives incorrect results?

Reply:
Yeah, not needed. That was just belt and braces. Removed!